Week-11/lightswitch/src/App.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import { useSelector , useDispatch} from 'react-redux';
+import {changeLight} from './action/action';
+import './App.css'
+
+const App = () => {
+  const myState = useSelector(state=>state.reducer);
+  const dispatch = useDispatch();
+  return (
+    <div className={myState ? 'dark' : 'lit'}>
```

Comment:
Make a variable like const themeClass = myState ? 'dark' : 'lit' and use it in view. Don't put logic into Views. You might need to separate them both in future.